src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -80,6 +80,12 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
 
 <PlatformContent includePath="performance/beforeNavigate-example" />
 
+<Note>
+  If you're using React, read the docs about how to set up your [React Router
+  integration]
+  (/platforms/javascript/guides/react/configuration/integrations/react-router/).
```

Comment:
I think the link text needs to all be on the same line to render properly

Reply:
Yup, you're totally right. I had it on the same line. Not sure why it's doing this...